tst/standard/digraph.tst

```diff
@@ -647,6 +647,14 @@ gap> RandomTournament(-1);
 Error, Digraphs: RandomTournament: usage,
 the argument <n> must be a non-negative integer,
 
+#  RandomLattice
+gap> RandomLattice(25);;
+gap> RandomLattice(1);
+<digraph with 1 vertex, 1 edge>
+gap> RandomLattice(-1);
+Error, no method found! For debugging hints type ?Recovery from NoMethodFound
+Error, no 1st choice method found for `RandomLattice' on 1 arguments
```

Comment:
Can you please add some tests that indicate approximately how good the distribution is obtained by `RandomLattice`. For example, something like:

~~~~
D := List([1 .. 100], x -> RandomLattice(7));
out := [];
for DD in D do 
  if ForAll(D, x -> not IsIsomorphicDigraph(x, DD)) then 
    Add(out, DD);
  fi;
od;
~~~~
Where you could replace the numbers 100 and 7 by something more sensible.